test/scraper/schleswig_holstein_landtag_scraper_overview_test.rb

```diff
@@ -58,6 +58,53 @@ def setup
 
   test 'get pdf url' do
     assert_equal 'http://www.landtag.ltsh.de/infothek/wahl18/drucks/2500/drucksache-18-2537.pdf', @scraper.extract_url(@blocks[15])
+    assert_equal 'http://www.landtag.ltsh.de/infothek/wahl18/drucks/2500/drucksache-18-2540.pdf', @scraper.extract_url(@blocks[0])
+  end
+
+  test 'get major paper' do
+    html = Nokogiri::HTML(File.read(Rails.root.join('test/fixtures/schleswigholstein_landtag_scraper_major.html')).force_encoding("WINDOWS-1252"))
+    table = @scraper.extract_table html
+    blocks = @scraper.extract_blocks table
+    paper = @scraper.extract_major_paper blocks[0]
+    assert_equal({
+      legislative_term: '17',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.